### DISCUSSION TITLE
Translated to French but it's 1.3.2

### DISCUSSION THREAD

**Discussion by grog12** [fraxiinus/ReplayBook#117] [General]
Hello,

I have translated ReplayBook in French, but the problem is that it is the 1.3.2 version as I have now gotten around to finishing the translation. Is there a way to import the translations that are still applicable/delete defunct ones rather than searching one by one to see if each line still exists in the newer version? Should I fork my 1.3.2 French translation anyway?

**Comment by fraxiinus** [comment]
Thanks for contributing a translation! 

If you want to update your translation to 1.4, I have a tool that can tell you the differences between two strings files. I can run your file through the tool and give you the results on what strings are missing or are not being used anymore. You can then use that comparison to update your translation for 1.4.  
You can attach your current file here and I'll reply with the comparison results.

Otherwise, you'll just have to to rename your translation to "fr.xml" and update your fork to 1.4. Then you should be good to do a pull request. I can remove any defunct lines later, and probably a good 80-90% of the application will still be translated.

**Reply by grog12** [reply]
https://github.com/grog12/ReplayBook/blob/master/Rofl.UI.Main/Resources/Strings/fr.xaml

I will try to updating it to 1.4 after

**Reply by fraxiinus** [reply]
Hey it looks like you're pretty up to date already! I fixed some xaml formatting errors but there was only one redundant string and one missing string. I removed the redundant one (ExecutableDefaultLocaleLabelText).

You are only missing "OpenNewWindow" from 1.4, the value for that is "Open in a New Window"

(I renamed it .txt to upload it to GitHub, just remove that)
[fr.xaml.txt](https://github.com/fraxiinus/ReplayBook/files/7585830/fr.xaml.txt)

**Reply by JerryLiew** [reply]
i used that script with the command that you provided in the translation wiki page, but it only shows the result of percentage of completion.Then i checked the source code and found the "-format  json"  argument.I guess it's better to mention it on the wiki page.

**Reply by fraxiinus** [reply]
Good idea, I'll update that (updated now!)
